### PR TITLE
Updates for Branch Migration

### PR DESCRIPTION
**Pull request by acidprime**
Clone of #7

### CODE REVIEW

**Comment by acidprime** [comment]
I modified a branch on learn to use this branch of the action

```diff
commit 53bae0207f5ae0216a3dbf738855e2a09029afb9
Author: Zachary Smith <acidprime@wallcity.org>
Date:   Wed Jan 26 10:57:47 2022 -0700

    Use branch for testing

diff --git a/.github/workflows/check-syntax.yml b/.github/workflows/check-syntax.yml
index 6ec4ac16a..f5400df48 100644
--- a/.github/workflows/check-syntax.yml
+++ b/.github/workflows/check-syntax.yml
@@ -15,7 +15,7 @@ jobs:
       - name: Download Markdown
         uses: actions/checkout@v2.0.0
       - name: Run Syntax Checks 
-        uses: hashicorp/learn-inspec@master
+        uses: hashicorp/learn-inspec@ks.adds-default-branch-env
         with:
           profile: 'github'
           markdown: ${{github.workspace}}

```
Successfully downloaded the action
sha

And was able to detect modified markdown files that differed from master with syntax errors, so as written this works on learn
error

@kendallstrautman

**Comment by kendallstrautman** [comment]
Fantasic! Thank you 🎉

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org' do
   gem 'inspec', '4.18.100'
   gem 'inspec-bin', '4.18.100'
-  gem 'kramdown', '2.1.0'
+  gem 'kramdown', '2.3.1'
```
Comment (acidprime) [inline Gemfile:4]: I have been getting a security warning on this so updating it as part of this change.

File: action.yml [hunk 2]
```diff
@@ -19,16 +19,21 @@ inputs:
...
+    description: 'The default branch in the target repository'
+    default: 'master'
+    required: false
 outputs:
   inspec-output:
     description: 'The output of inspec'
 runs:
   using: 'docker'
-  image: 'docker://hashieducation/inspec' 
+  image: 'docker://docker.mirror.hashicorp.services/hashieducation/inspec:ks.adds-default-branch-env'
```
Comment (acidprime) [inline action.yml:31]:
```suggestion
  image: 'docker://docker.mirror.hashicorp.services/hashieducation/inspec:master'
```
Need to update this after we push to docker hub and sync to mirror and before we merge this PR.

**Comment by acidprime** [comment]
This is related to https://github.com/hashicorp/tutorials/pull/6

**Pull request merged**